Block/JsProductPage.php

```diff
@@ -181,7 +181,7 @@ public function isBoltProductPage()
         // all that remains is to check if this product has the ppc attribute or not.
         $attributes = $this->getProduct()->getAttributes();
         foreach ($attributes as $attribute) {
-            if ($attribute->getName() == 'bolt_ppc') {
+            if ($attribute->getName() == 'bolt_ppc' && $this->getProduct()->getBoltPpc()) {
```

Comment:
why do we need two conditions here? can the whole logic be simplified, reduced to one condition with some additional change elsewhere?

Reply:
@danac-gs Good catch. I've updated the PR. Could you please re-check it?